packages/aragon-cli/src/commands/apm_cmds/util/generate-artifact.js

```diff
@@ -129,14 +129,13 @@ function sanityCheck(
   modulePath,
   artifact
 ) {
-  const { environments, contractPath } = artifact
-  const { appName, registry, network } = environments[0]
```

Comment:
It was failing here because `environments` is not an array.
That being said `environments['default']` does not feel alright either.